helm-chart/splunk-connect-for-kubernetes/charts/splunk-kubernetes-logging/templates/configMap.yaml

```diff
@@ -73,7 +73,7 @@ data:
     <source>
       @id containers.log
       @type tail
-      @label @SPLUNK
+      @label @CONCAT
```

Comment:
Why are these changes included here?
They are already in develop.
Please rebase and squash commits.